library/swift/BUILD

```diff
@@ -1,9 +1,11 @@
-load("@envoy_mobile//bazel:swift_static_framework.bzl", "swift_static_framework")
+load("@build_bazel_rules_apple//apple:ios.bzl", "ios_static_framework")
+load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
+load("//bazel:swift_header_collector.bzl", "swift_header_collector")
 
 licenses(["notice"])  # Apache 2
 
-swift_static_framework(
-    name = "ios_framework",
+swift_library(
+    name = "lib",
```

Comment:
Any particular reason for this name (e.g. like does it align with some convention or is it required by some rule)? If not, as a nit, we might want to pick something a little more explicit.